src/uniprotkb/utils/__tests__/__mocks__/databaseInfo.ts

```diff
@@ -281,7 +281,7 @@ const databaseInfo: DatabaseInfo = [
     name: 'CAZy',
     displayName: 'CAZy',
     category: 'PFAM',
-    uriLink: 'http://www.cazy.org/fam/%id.html',
+    uriLink: 'http://www.cazy.org/%id.html',
```

Comment:
Make sure to update the "retrieved" comment at the top so that anyone checking this in the future can easily see how old this mock is

Reply:
Thanks for reminding. I updated it